packages/braid-design-system/src/lib/components/private/MaxLines/MaxLines.css.ts

```diff
@@ -10,6 +21,11 @@ export const base = style([
   {
     textOverflow: 'ellipsis',
     whiteSpace: 'nowrap',
+    selectors: {
+      [`${descenderCropFixForWebkitBox} &`]: {
```

Comment:
Not that we allow nesting our typography components, but maybe a more full proof way is always ensure the `base` class is a direct child of the parent.
```suggestion
      [`${descenderCropFixForWebkitBox} > &`]: {
```